alembic/env.py

```diff
@@ -0,0 +1,83 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from city.models import *
+from temperature.models import *
```

Comment:
Similar to the previous comment, using `from temperature.models import *` can cause namespace issues and reduce code readability. Consider importing only the specific models you need.